ms_rewards.py

```diff
@@ -782,40 +771,40 @@ def get_point_total(pc=False, mobile=False, log=False):
     # wait_until_visible(By.XPATH, '//*[@id="flyoutContent"]', 10)  # check for loaded point display
 
     # TODO add a scroll to obj here
-    if not wait_until_visible(By.CLASS_NAME, 'pcsearch', 10):  # if object not found, return False
-        return False
+    wait_until_visible(By.CSS_SELECTOR, 'div#userPointsBreakdown > div > div:nth-of-type(2) > h2', 10) # if object not found, return False
     # returns None if pc search not found
     # pcsearch = browser.find_element_by_class_name('pcsearch')
     # if pcsearch.location_once_scrolled_into_view is None:  # property causes pc search to be scrolled into area
     #     return False
 
     try:
+        time.sleep(3)
         current_point_total = list(map(
-            int, browser.find_element_by_class_name('credits2').text.split(' of ')))[0]
+            int, browser.find_element_by_css_selector('#userBanner > mee-banner > div > div > div > div.info-columns > div:nth-child(3) > mee-banner-slot-4 > mee-rewards-user-status-item > mee-rewards-user-status-dailypoint > div > div > div > div > div > p.bold.number.margin-top-1 > mee-rewards-counter-animation > span > b').text))[0]
```

Comment:
Big CSS locators Iike this are very unstable and require frequent maintenance.
Please update an xpath or a better locator.

Reply:
I adjusted. I create Git Pull right away. Future requests will only be made via direct ID allocation or XPATH. Now use the Katalon Recorder :) Makes work much :)

Reply:
XPATHs like this are also unstable and will need frequent maintenance. Please check this [link](https://devhints.io/xpath) to get a better understanding of writing better XPATH.